components/resources/commands.grdp

```diff
@@ -269,7 +269,7 @@
     Customize touch bar
   </message>
   <message name="IDS_IDC_TOGGLE_JAVASCRIPT_APPLE_EVENTS">
-    Toggle javascript apple events
+    Toggle JavaScript apple events
```

Comment:
Apple Events should also be capitalized according to https://source.chromium.org/chromium/chromium/src/+/main:chrome/app/generated_resources.grd;l=14153